integration-tests/src/test/java/oracle/kubernetes/operator/ItManagedCoherence.java

```diff
@@ -373,4 +405,62 @@ private ExecResult clearCache() throws Exception {
     return result;
   }
 
+  private void restartCluster(String domainUid) throws Exception {
```

Comment:
can you make this method generic, so that if we have to use from other test classes it can be used? does the other multi cluster test you are debugging needs to restart the cluster as well?

Reply:
restarting the cluster resolves the issue, but not sure it is the right solution. Not calling this method now, but did not want to delete it in case we decide that is what needs to be done.

Reply:
Deleted the method. Will add it later if need to.